### ISSUE TITLE
Local Setup Guide for Development and Testing

### ISSUE THREAD

**Issue by HlexNC** [Nutlope/aicommits#236]
### Feature request

Provide detailed instructions on setting up and running the `aicommits` project locally for development and testing purposes, without the need for global npm installation.

### Why?

Currently, the only provided method for running `aicommits` is via global npm installation. This approach is suitable for end-users, but it's not ideal for developers who wish to contribute to the project or test it locally in a controlled environment.

A local setup guide would simplify the process for developers looking to contribute to the project, enabling them to iterate and test changes more effectively before submitting pull requests. This is especially useful for open-source projects where the contributors may range from beginners to experienced developers.

Moreover, having a clear, local setup guide helps maintain consistency in development environments, reducing the chances of inconsistencies and "it works on my machine" issues.

### Additional context

Including detailed instructions on prerequisites (required software and versions), environment setup, installation of dependencies, and execution commands in the project's `README.md` would be beneficial.

**Comment by HlexNC** [comment]
I would like to add that I've already tried setting up the project locally by performing the following steps:

- Cloning the repository with `git clone`.
- Installing the dependencies with `npm install`.
- Attempting to build the project with `npm run build`.
- Trying to link the package locally with `npm link`.

While these are standard steps for many Node.js projects, I found that this approach didn't work for `aicommits`. A more detailed setup guide tailored to this project's specific needs would therefore be incredibly helpful. This will not only help me but also any other developers who want to contribute to this project in the future.
